requirements.txt

```diff
@@ -30,23 +30,23 @@ kubernetes==6.0.0
 MarkupSafe==1.0
 model-mommy==1.6.0
 openapi-codec==1.3.2
-psycopg2==2.7.4
-py==1.5.3
+psycopg2==2.7.5
```

Comment:
we should probably change this to psycopg2-binary

Reply:
Or change this to `psycopg2==2.7.5 --no-binary psycopg2`? Which do you think? I'm not sure about psycopg2-binary...

Reply:
for now it's fine but when it hits 2.8 we'll have to decide: http://initd.org/psycopg/articles/2018/02/08/psycopg-274-released/